packages/contracts/ethpm.js

```diff
@@ -0,0 +1,8 @@
+{
+    "package_name": "@artonomous/contracts"
+    "version": "0.1.0",
+    "dependencies": {
+        "zeppelin": "1.3.0",
+        "tokens": "1.0.0"
+    }
+}
```

Comment:
I'm not familiar with how ethpm works here. Is this needed atm to make our contracts work? I haven't run the full setup recently.